src/main/java/org/openrewrite/java/template/processor/RefasterTemplateProcessor.java

```diff
@@ -466,6 +467,10 @@ private String recipeDescriptor(JCTree.JCClassDecl classDecl, String defaultDisp
                             }
                         }
                         break;
+                    } else if ("tech.picnic.errorprone.refaster.annotation.OnlineDocumentation".equals(annotationFqn)) {
+                        if (annotation.getArguments().isEmpty()) {
```

Comment:
It seems rare / unused to have any value here ; they mostly rely on [the default value](https://github.com/PicnicSupermarket/error-prone-support/blob/8f64489fa0dded9255d20c227d0810af1cf52493/refaster-support/src/main/java/tech/picnic/errorprone/refaster/annotation/OnlineDocumentation.java#L44-L48), which isn't immediately easily accessible here for as much as I can tell. Hence why I went for the easiest approach to have some links to the docs already, which we can refine later.